app/api/models/video.py

```diff
@@ -4,7 +4,6 @@
 def add_models_to_namespace(api_namespace):
     api_namespace.models[add_video_model.name] = add_video_model
     api_namespace.models[add_yt_video_model.name] = add_yt_video_model
-    api_namespace.models[yt_video_section_field.name] = yt_video_section_field
```

Comment:
why you remove this? is it related to this pr?

Reply:
@annabauza This is related to the #86. As we have removed sections from youtube payload, it is no more needed. Somehow forgot to remove there, so did here.

Reply:
add to the description here please :) thanks!